proto/qdrant/cloud/monitoring/v1/monitoring.proto

```diff
@@ -67,6 +67,34 @@ service MonitoringService {
       field_expression: "cluster_id"
     };
   }
+  // Gets the inference token usage metrics aggregated per inference model for a cluster.
+  // Required permissions:
+  // - read:clusters
+  rpc GetClusterInferenceMetrics(GetClusterInferenceMetricsRequest) returns (GetClusterInferenceMetricsResponse) {
+    // permissions
+    option (common.v1.permissions) = "read:clusters";
+    // gRPC Gateway REST call
+    option (google.api.http) = {get: "/api/monitoring/v1/accounts/{account_id}/cluster/{cluster_id}/metrics/inference"};
+    // log fields
+    option (common.v1.log_fields) = {
+      name: "cluster_id"
+      field_expression: "cluster_id"
+    };
+  }
+  // Gets the inference token usage metrics for a specific inference model in a cluster.
+  // Required permissions:
+  // - read:clusters
+  rpc GetClusterInferenceModelMetrics(GetClusterInferenceModelMetricsRequest) returns (GetClusterInferenceModelMetricsResponse) {
+    // permissions
+    option (common.v1.permissions) = "read:clusters";
+    // gRPC Gateway REST call
+    option (google.api.http) = {get: "/api/monitoring/v1/accounts/{account_id}/cluster/{cluster_id}/metrics/inference/{inference_model_id}"};
+    // log fields
+    option (common.v1.log_fields) = {
+      name: "cluster_id"
+      field_expression: "cluster_id"
+    };
+  }
```

Comment:
I would prefer to merge into each other, keep 1 and make the model an optional field, based on the result, this is OK as well

Reply:
should be good IMO. Also could you check the PR description? There I put two sample requests and reponses with/without inference_model_id.